code/obj/flock/structure/relay.dm

```diff
@@ -49,7 +46,6 @@
 
 /obj/flock_structure/relay/disposing()
 	..()
-	//crew destroyed it, let them call shuttle
```

Comment:
useful comment

Reply:
dubiously useful, intent is declared through `disposing` override. e.g. "This does not exist anymore, allow it"